stl/src/multprec.cpp

```diff
@@ -170,7 +170,6 @@ void __CLRCALL_PURE_OR_CDECL _MP_Rem(
         }
         // D5: [Test remainder.]
         if (k != 0) { // D6: [Add back.]
-            --qh;

```

Comment:
No change requested, noting for other reviewers: This is safe because `qh` is a loop-local variable on line 131.